### PR TITLE
Fix minor problems in docs

### PR DESCRIPTION
**Pull request by lirc572**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -111,7 +111,7 @@ Different sections of the application window will be referred to by the names de
 
 Although most of the buttons you see on the screen are clickable, the UI has been designed primarily to be navigated using the command line interface.
 
-Users can enter commands into the command box and press `ENTER` to execute it. The result box will be updated with information on whether the command has been successfully executed. Depending on the command executed, the main panel may also be updated based on the command executed.
+Users can enter commands into the command box and press `ENTER` to execute them. The result box will be updated with information on whether the command has been successfully executed. Depending on the command executed, the main panel may also be updated based on the command executed.
```
Comment (vevek) [inline docs/UserGuide.md:114]: change them to it

File: docs/UserGuide.md [hunk 2]
```diff
@@ -159,13 +159,13 @@ Parameters refer to information that is to be included as an input to a command.
...
-To achieve the best possible experience, avoid adding excessively long text to CoLAB. At small screen resolutions and with unusually long parameters, text may be truncated to  fit in the main panel. While we do not expect such occurrences to happen in day to day regular use, we have made accommodations for it. To view the rest of the truncated text, simple hover over the relevant section and scroll sideways, either with a trackpad or shift-scroll if you are using a mouse with a scroll wheel.
+To achieve the best possible experience, avoid adding excessively long text to CoLAB. At small screen resolutions and with unusually long parameters, text may be truncated to  fit in the main panel. While we do not expect such occurrences to happen in day-to-day regular use, we have made accommodations for it. To view the rest of the truncated text, simply hover over the relevant section and scroll sideways, either with a trackpad or shift-scroll if you are using a mouse with a scroll wheel.
 </div>
 
 ##### `ADDRESS`
 
 * The address of a contact.
-* Addresses can take any values, but it should not be blank.
+* An addresses can take any value, but it should not be blank.
```
Comment (vevek) [inline docs/UserGuide.md]: address

File: docs/UserGuide.md [hunk 3]
```diff
@@ -663,7 +663,7 @@ Parameters:
 
 Examples:
 
-* `updateE 1 i/1 d/Project Meeting on/24-04-2021 w/Y` updates the first event of the first project to be weekly event with new description `Project Meting` starting on `24-04-2021`.
+* `updateE 1 i/1 d/Project Meeting on/24-04-2021 w/Y` updates the first event of the first project to be weekly event with a new description `Project Meeting` starting on `24-04-2021`.
```
Comment (vevek) [inline docs/UserGuide.md]: to be a weekly event

File: docs/UserGuide.md [hunk 4]
```diff
@@ -717,10 +717,10 @@ A groupmate can have any number of roles (including 0).
 
 Examples:
 
-* `addG 1 n/John Doe` adds a groupmate with name `John Doe` to the first project.
-* `addG 2 n/Betsy Crowe r/developer` adds a groupmate with name `Betsy Crowe` and role `developer` to the second project.
+* `addG 1 n/John Doe` adds a groupmate with the name `John Doe` to the first project.
+* `addG 2 n/Betsy Crowe r/developer` adds a groupmate with the name `Betsy Crowe` and role `developer` to the second project.
```
Comment (vevek) [inline docs/UserGuide.md]: `Betsy Crowe` and the role

File: docs/UserGuide.md [hunk 5]
```diff
@@ -769,13 +769,13 @@ Example:
 
 ### **4.2 Contacts**
 
-Contact are used in CoLAB to store contact information about a person, to help you keep track of people you contact frequently.
+Contact is used in CoLAB to store contact information about a person, to help you keep track of people you contact frequently.
```
Comment (vevek) [inline docs/UserGuide.md]: Contacts are

File: docs/UserGuide.md (hunk 1 above)
Comment (lirc572) [reply docs/UserGuide.md:114]: them refers to commands I think

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -148,7 +148,7 @@ This section describes some noteworthy details on how certain features are imple
 
 ### View Projects Feature
 
-This section explains the implementation of the View Project feature. The implementation of other commands that opens panels or tabs are similar.
+This section explains the implementation of the View Project feature. The implementations of other commands that open panels or tabs are similar.
```
Comment (vevek) [inline docs/DeveloperGuide.md]: implementation?

**Review by vevek — DISMISSED** [review]
LGTM

**Review by vevek — APPROVED** [review]
LGTM

**Pull request merged**